src/view-compiler.js

```diff
@@ -158,7 +158,7 @@ export class ViewCompiler {
       return this._compileElement(node, resources, instructions, parentNode, parentInjectorId, targetLightDOM);
     case 3: //text node
       //use wholeText to retrieve the textContent of all adjacent text nodes.
-      let expression = resources.getBindingLanguage(this.bindingLanguage).inspectTextContent(resources, node.wholeText);
+      let expression = resources.getBindingLanguage(this.bindingLanguage).inspectTextContent(resources, node.wholeText || node.data);
```

Comment:
I don't see a way to determine what text accessor property is available without introducing unnecessary boilerplate or performance penalty (e.g. with `hasOwnProperty` on the `node`). 

The safer way could be: `node.wholeText !== undefined ? node.wholeText : node.data`.

This is the issue on jsdom having it unimplemented: https://github.com/tmpvar/jsdom/issues/1397